planemo/commands/cmd_shed_diff.py

```diff
@@ -53,7 +59,7 @@ def cli(ctx, path, **kwds):
     def diff(realized_repository):
         working = tempfile.mkdtemp(prefix="tool_shed_diff_")
         try:
-            shed.diff_in(ctx, working, realized_repository, **kwds)
+            return shed.diff_in(ctx, working, realized_repository, **kwds)
```

Comment:
only return in one place, returning (I'm assuming) an error code, or None. Maybe better to explicitly return None/-1?  Sorry, it's late, tipsy code review is probably not the best ;)